src/app/team/social-sharing/social-sharing.component.html

```diff
@@ -0,0 +1,9 @@
+<a *ngIf="!!githubUrl" mat-button [href]="githubUrl">
+  <mat-icon [ngStyle]="{'color': '#6f066f'}" fontSet="fa" fontIcon="fa-github"></mat-icon>
```

Comment:
Please move the colors into scss and use classes.

Reply:
There are 3 `ngIf` - just a thought if this can be done without `ngIf` by handling the transformation in js.